src/charm.py

```diff
@@ -255,7 +277,16 @@ def ready_to_start(self) -> bool:
         return True
 
     def get_secret(self, scope: str, key: str) -> Optional[str]:
-        """Get TLS secret from the secret storage."""
+        """Get TLS secret from the secret storage.
+
+        Args:
+            scope: whether this secret is for a `unit` or `app`
+            key: the secret key name
+
+        Returns:
+            String of key value.
+            None if non-existent key
+        """
         if scope == "unit":
             return self.unit_peer_data.get(key, None)
```

Comment:
`self.unit_peer_data.get(key)` behaves the same as `self.unit_peer_data.get(key, None)`